ontology/yaml/resources/METERS/entity_types/EM.yaml

```diff
@@ -28,6 +28,17 @@ EM_PWM_PVCM:
   - EM
   - PWM
   - PVCM
+  opt_uses:
+  - reactive_energy_accumulator
+  - apparent_energy_accumulator
+
+EM_PWM_AVCM:
```

Comment:
Cannot have a duplicate, see line 43.

Reply:
done sir, although i don't know how this could happen. probably through the auto merge...